From 62e57de900b55b8c5b2a0b81095f7200c2af4c74 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Thu, 7 Feb 2019 21:22:11 -0700
Subject: [PATCH 5/7] legacy: Label some more sysfs_net

* avc: denied { read } for name="type" dev="sysfs" ino=29740
  scontext=u:r:priv_app:s0:c512,c768 tcontext=u:object_r:sysfs:s0
  tclass=file permissive=0
* ino=29740 -> /sys/class/net/wlan0/type

Change-Id: Ie0c606049a064e189386cdf434a649b93c9c014f
---
 common/file_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/file_contexts b/common/file_contexts
index 120eaa9..76d20e8 100644
--- a/common/file_contexts
+++ b/common/file_contexts
@@ -332,6 +332,7 @@
 /sys/devices/f9200000\.*/power_supply/usb(/.*)?                     u:object_r:sysfs_usb_supply:s0
 /sys/devices/msm_dwc3/power_supply/usb(/.*)?                        u:object_r:sysfs_usb_supply:s0
 /sys/devices/msm_otg/power_supply/usb(/.*)?                         u:object_r:sysfs_usb_supply:s0
+/sys/devices(/platform)?/msm_sdcc\.[0-9]/mmc_host/mmc[0-9]/mmc[0-9]:[a-f0-9]+/mmc[0-9]:[a-f0-9]+:[a-f0-9]+/net(/.*)?  u:object_r:sysfs_net:s0
 /sys/devices(/platform)?/soc/[a-z0-9]+\.qcom,spmi/spmi-[0-9]/spmi0-0[0-9]/[a-z0-9]+\.qcom,spmi:qcom,[a-z0-9]+@[0-9]:qcom,qpnp-smb[0-9]+/power_supply/usb(/.*)?        u:object_r:sysfs_usb_supply:s0
 /sys/devices(/platform)?/soc/[a-z0-9]+\.qcom,spmi/spmi-[0-9]/spmi0-0[0-9]/[a-z0-9]+\.qcom,spmi:qcom,[a-z0-9]+@[0-9]:qcom,qpnp-smb[0-9]+/power_supply/pc_port(/.*)?    u:object_r:sysfs_usb_supply:s0
 /sys/devices(/platform)?/soc/[a-z0-9]+\.i2c/i2c-[0-9]+/[0-9]+-[a-z0-9]+/[a-z0-9]+\.i2c:qcom,[a-z0-9]+@[a-z0-9]:qcom,smb[0-9]+-charger@[0-9]+/power_supply/parallel(/.*)?    u:object_r:sysfs_usb_supply:s0
-- 
2.17.1

